<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>centralWidget</class>
 <widget class="QWidget" name="centralWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>309</width>
    <height>488</height>
   </rect>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QLabel" name="titleLabel">
     <property name="text">
      <string>Speech to Text Converter</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" name="raido">
     <item>
      <widget class="QRadioButton" name="YoutubeRadio">
       <property name="text">
        <string>With Url</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QRadioButton" name="FileRadio">
       <property name="text">
        <string>With File</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="Check">
     <item>
      <widget class="QCheckBox" name="Subtitle">
       <property name="text">
        <string>Subtitle Result</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QCheckBox" name="Text">
       <property name="text">
        <string>Text  Result</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QLineEdit" name="inputField">
     <property name="placeholderText">
      <string>Enter the YouTube URL</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QPushButton" name="convertButton">
     <property name="text">
      <string>Convert to Text</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QProgressBar" name="progressBar">
     <property name="styleSheet">
      <string notr="true">QProgressBar::chunk { background: rgb(0, 0, 255); }</string>
     </property>
     <property name="maximum">
      <number>0</number>
     </property>
     <property name="value">
      <number>0</number>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QTextEdit" name="outputText"/>
   </item>
   <item>
    <layout class="QHBoxLayout" name="buttonLayout">
     <item>
      <widget class="QPushButton" name="saveButton">
       <property name="text">
        <string>Save Result</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="copyButton">
       <property name="text">
        <string>Copy to Clipboard</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
